source/installer.py

```diff
@@ -432,6 +455,7 @@ def createPortableCopy(destPath,shouldCopyUserConfig=True):
 	tryCopyFile(os.path.join(destPath,"nvda_noUIAccess.exe"),os.path.join(destPath,"nvda.exe"))
 	if shouldCopyUserConfig:
 		copyUserConfig(os.path.join(destPath,'userConfig'))
+	removeOldLibFiles(destPath,rebootOK=False)

```

Comment:
Why is `rebootOK=False` here, and `rebootOK=True` in `def install`?

Reply:
Because  usually for portable copies it is impossible to delete after reboot as you need admin  privilege to set this. However, there is no error if it doesn't work, so I've now allowed it anyway. Some people always running as admin this can work for. Plus, this also ensures that files remain renamed to temp files even if   they can't be deleted on reboot, which keeps them out of the way of future installs.